plugins/ccgx/fu-ccgx-dmc-device.c

```diff
@@ -684,6 +682,9 @@ fu_ccgx_dmc_device_setup(FuDevice *device, GError **error)
 	fu_device_set_version(FU_DEVICE(self), version);
 	fu_device_set_version_raw(FU_DEVICE(self), version_raw);
 	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_UPDATABLE);
+
+	if (dock_id.custom_meta_data_flag > 0)
+		fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_SIGNED_PAYLOAD);
```

Comment:
does that mean we can add `FWUPD_DEVICE_FLAG_UNSIGNED_PAYLOAD` if the opposite is true?

Reply:
> does that mean we can add FWUPD_DEVICE_FLAG_UNSIGNED_PAYLOAD if the opposite is true?

Yes. your are right. I will add FWUPD_DEVICE_FLAG_UNSIGNED_PAYLOAD if the opposite is true?

Reply:
v